opal/mca/base/mca_base_var.h

```diff
@@ -696,10 +696,12 @@ OPAL_DECLSPEC int mca_base_var_build_env(char ***env, int *num_env, bool interna
 typedef enum {
     /* Dump human-readable strings */
     MCA_BASE_VAR_DUMP_READABLE = 0,
+    /* Dump human-readable strings, with color where supported */
+    MCA_BASE_VAR_DUMP_READABLE_COLOR = 1,
```

Comment:
Can we add this enum at the end of the list?  I don't *think* that there are going to be any backwards-compatible issues with re-numbering these particular enums, but let's not tempt fate unless we have to.